edd-license-free-download.php

```diff
@@ -41,10 +34,22 @@ public static function init() {
 
 		add_shortcode( 'edd_lfd', array( __CLASS__, 'shortcode_form' ) );
 
+		add_action( 'plugins_loaded', array( __CLASS__, 'licensing' ), 20 );
+
 		add_filter( 'edd_settings_sections_extensions', array( __CLASS__, 'register_subsection' ) );
 		add_filter( 'edd_settings_extensions', array( __CLASS__, 'settings_page' ) );
 	}
 
+	/**
+	 * Use the EDD built in license handler.
+	 *
+	 * @since 1.0.1
+	 */
+	public static function licensing() {
+		if ( class_exists( 'EDD_License' ) ) {
+			$license = new EDD_License( __FILE__, 'License Free Download', '1.0.1', 'Sandhills Development, LLC' );

```

Comment:
```suggestion
			$license = new EDD_License( __FILE__, 'License Free Download', '1.0', 'Sandhills Development, LLC' );

```

Can you keep this at `1.0` for now? We'll increment it prior to releasing the update.